### ISSUE TITLE
Session health and logout from smart-faq

### ISSUE THREAD

**Issue by git-toni** [kiwicom/smart-faq#46]
Open questions:
- Should we allow the user to logout of kiwi.com completely from the smart-faq component?
- Should smart-faq check the "health" of the user's session/token and kill their session if the verification is unsuccessful?

**Comment by jalbertsr** [comment]
- Yes to the first question, because as a user I would understand kiwi.com frontend and smart-faq as a one. 
- Why should we not kill a session if the verification is unsuccessful? Are you talking about the case they come already logged from kiwi.com?

**Comment by git-toni** [comment]
> Yes to the first question, because as a user I would understand kiwi.com frontend and smart-faq as a one.

Cool, then we need some options in the UI for that :) I was asking because maybe the Logout shouldn't be provided through the smart-faq component directly, since it is meant to be a sub-component of the whole kiwicom after all.

> Why should we not kill a session if the verification is unsuccessful? Are you talking about the case they come already logged from kiwi.com?

Actually it may be irrelevant whether the user is already logged-in from kiwicom or not. I raised the question because I figured we may need to remove -session- data from their redux store in case we do verify and it doesn't pass. But as far as I saw, their session info is all Cookie-based, so no JS store involved.

Removal of cookie upon wrong session info is not implemented in smart-faq yet, so I suggest we raise that as another issue.
